syntax = "proto3";

package pb;

message Message {
    string name = 0;     // Sender's name
    string receiver = 1; // Receiver's name
    string text = 2;     // Text data
    bytes file_data = 3; // File data
}
